content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-12-0.md

```diff
@@ -0,0 +1,19 @@
+---
+title: v1.12.0 Armory Halyard
+toc_hide: true
+version: 01.12.00
+description: Release notes for Armory Halyard v1.12.0
+---
+
+## 05/07/2021 Release Notes
```

Comment:
We should ask @jorgebee65 and @rvazquezglez for something we can use to highlight the fact that Docker registries can use regex filters now

Reply:
Or maybe @brian-armory already has a snippet for this, I don't remember

Reply:
taken care of